### ISSUE TITLE
5.0.9 to 5.0.14 Disk Scanning fails for leading space in lsblk values

### ISSUE THREAD

**Issue by Hooverdan96** [rockstor/rockstor-core#2907]
Thanks to user cellisten on the community forum, there seems to be an issue when the model name contains a space within. Usual nomenclature found on model names usually does not contain any spaces.

https://forum.rockstor.com/t/scanning-disks-not-working/9781

There seems to already be a hint on how to address it, but that needs confirmation that it won't have an impact on other scenarios. From the message thread:

> I did some refactoring of the generation of the blk_dev_properties to separate the line in a manner where spaces don’t really matter and the strip is done differently:

```
line = [ item.strip() for item in re.split(r'["=]', line) ]
line = [ item for i, item in enumerate(line,-1) if i %3 != 0 ]
# Device information built from each lsblk line in turn.
blk_dev_properties: dict = {
key.lower() if key != "TRAN" else "transport":
value if value != "" else None for key, value in zip(line[::2], line[1::2])
}
```

**Comment by phillxnet** [comment]
@Hooverdan96 Thanks for creating this. Linking for context to a similar fix re lsblk VENDOR white space sensitivity we fixed in  5.0.11-0:

Issue: 5.0.9-0 & 5.0.10-0 lsblk whitespace only values - not enough values to unpack #2853
PR: 5.0.9-0 & 5.0.10-0 lsblk whitespace only values - not enough values to unpack #2853 #2855

**Comment by cellisten** [comment]
Not sure if I sent the pull request correctly, my first time contributing here. Can't seem to add the issue directly to the pull request but I'm guessing it's because I don't have permission to do so

**Comment by phillxnet** [comment]
@cellisten Re:

> Not sure if I sent the pull request correctly

It looks OK at this end.

> Can't seem to add the issue directly to the pull request

I did it earlier, all I did was add a link into the PR text to this issue. I'll edit the PR text a little too as forward slashes can have strange effects in git messages/branch-names etc. We also need the PR and Issue to have a correlated name and to container (in the PR) the issue it relates to. We auto generate the changelog and present it in the Web-UI so there are some constraints.
